src/igtcloud/client/services/entities_service.py

```diff
@@ -359,15 +360,13 @@ def get_s3_file_metadata(file: File, action: str):
     return s3.head_object(Bucket=creds.bucket, Key=file.key)
 
 
-def process_file(study: RootStudy, file: File):
-    if study.study_type in ['AnnotationStudy'] and file.type in ['.dcm', '.fxd', '']:
-        from . import action_service
-        from .action.model.preprocessing_request import PreprocessingRequest
-        request = PreprocessingRequest(hospital_id=study.institute_id,
-                                       study_id=study.study_database_id,
-                                       patient_id=getattr(study, 'patient_database_id', None),
-                                       uploaded_path=file.file_name)
-        action_service.post_preprocess_file(request)
+def file_upload_completed(study: RootStudy, file: File):
+    from . import entities_service
```

Comment:
You don't need these local imports here I expect, because you are in the entities_service.py file

Reply:
Removed local import.